Documentation/CodeSnippets/TypoScriptSyntax/OperatorAssignment.rst.txt

```diff
@@ -0,0 +1,31 @@
+.. Automatic screenshot: Remove this line if you want to manually change this file
+
+.. code-block:: typoscript
+   :caption: Extension examples, file Configuration/TypoScript/Syntax/Assignment/setup.typoscript
+
+   # Identifier "myIdentifier" is set to the value "foo"
+   myIdentifier = foo
+
+   # Identidfier path "myIdentifier.mySubIdentifier" is set to the value "foo"
+   myIdentifier.mySubIdentifier = foo
+
+   # "myIdentifier.mySubIdentifier" it set to the value "foo",
+   # but is immediately overwritten to value "bar"
+   myIdentifier.mySubIdentifier = foo
+   myIdentifier.mySubIdentifier = bar
+
+   # Same as above, value of "myIdentifier.mySubIdentifier" is "bar"
+   myIdentifier.mySubIdentifier = foo
+   myIdentifier {
+      mySubIdentifier = bar
+   }
+
+   # Value assignments are not comment-aware, "#", "//" and "/*" after a
+   # "=" operator do not start a comment. The value of identifier
+   # "myIdentifier.mySubIdentifier" is "foo // not a comment"
+   myIdentifier.mySubIdentifier = foo // not a comment
```

Comment:
actually there exists even a functional meaning i.e. using `title // subtitle // alt ...` to assign fall backs.  
Not sure if this should be  mentioned or linked here.

Reply:
nope, that would be title || subtitle || alt

Reply:
have a look at ATagTitle  to see what I meant:
https://docs.typo3.org/m/typo3/reference-typoscript/11.5/en-us/MenuObjects/Tmenuitem/Index.html

Reply:
David is correct, it is "//". But this is about comments, so I don't think we should mention the fallback of fields as it does not belong here.